### PR TITLE
Release 3.3.0

### PR DESCRIPTION
**Pull request by lucas-koehler**

### CODE REVIEW

File: static/current-version.js [hunk 1]
```diff
@@ -1,2 +1,2 @@
-export const currentVersion = 'currentVersion';
-export const nextVersion = 'nextVersion';
+export const currentVersion = '3.3.0';
+export const nextVersion = undefined;
```
Comment (sdirix) [inline static/current-version.js]: This does not need to be adapted, the file is automatically touched during the build process.

File: content/docs/deprecated/available-actions.mdx [hunk 2]
```diff
@@ -18,7 +18,7 @@ All actions might either be imported directly or via the Actions utility.
 
 ## `init(data: any, schema: JsonSchema, uischema: UISchemaElement, options: InitActionOptions | AJV.Ajv)`
 
-<ApiLink link='core/interfaces/initaction.html' title='API' />
+<ApiLink link='core/interfaces/Actions.InitAction.html' title='API' />
```
Comment (sdirix) [inline content/docs/deprecated/available-actions.mdx:21]: Do we have to do these adaptions manually every time we move the generated API?

Comment (lucas-koehler) [reply content/docs/deprecated/available-actions.mdx:21]: I guess so. But I think as this doc is in the deprecated section, we can also just remove it at some point. JF 2.5.x was a very long time ago after all.

File: static/current-version.js (hunk 1 above)
Comment (lucas-koehler) [reply static/current-version.js]: makes sense, I'll revert it

File: content/docs/deprecated/available-actions.mdx (hunk 2 above)
Comment (lucas-koehler) [reply content/docs/deprecated/available-actions.mdx:21]: Alternatively we could just remove the API links in the future but as I already fixed them now, I think we can leave it like this for now.

**Review by sdirix — APPROVED** [review]

**Pull request merged**